dandi/metadata.py

```diff
@@ -143,11 +148,11 @@ def timedelta2duration(delta):
     s = "P"
     if delta.days:
         s += f"{delta.days}D"
-    if delta.seconds or delta.microseconds:
+    elif delta.seconds or delta.microseconds:
```

Comment:
This should stay as `if`, in case `delta` has both `days` and `seconds`/`microseconds`.

Reply:
oh, i see, thanks!